src/main/resources/application.properties

```
@@ -1,4 +1,5 @@
 spring.datasource.url= jdbc:mysql://localhost/ongvida?createDatabaseIfNotExist=true&serverTimezone=UTC
 spring.datasource.username=root
 spring.datasource.password=root
-sad
+spring.jpa.hibernate.ddl-auto= create
```

tira o create e coloca tudo como update pra evitar a base de dados ficar apagando toda vez que roda de novo o projeto
```suggestion
spring.jpa.hibernate.ddl-auto= update
```